docs/articles/documentation/recipes/debug-in-webstorm.md

```diff
@@ -0,0 +1,120 @@
+
+Add a `debug` script to the `scripts` section in `package.json`. This script contains a command that runs TestCafe and specifies the target browser.
+
+```json
+{
+    "scripts": {
+        "debug": "testcafe $NODE_DEBUG_OPTION chrome"
+    },
+
+    "devDependencies": {
+        "testcafe": "^x.y.z"
+    }
+}
+```
+
+The `$NODE_DEBUG_OPTION` placeholder marks a place where Node.js debugger options are inserted before execution.
+
+Path to test files will be specified in the debug configuration at the next steps.
+
+## Step 3 - Create a Debug Configuration
+
+Open WebStorm. If no build or run configuration is set up at the moment, click **Add Configuration** in the top-right corner.
+
+![Click Add Configuration](../../images/webstorm/press-add-configuration.png)
+
+Otherwise, open the configuration drop-down menu and select **Edit Configurations...**.
+
+![Edit Configurations](../../images/webstorm/edit-configurations.png)
+
+This invokes the **Run/Debug Configurations** dialog.
+
+Click the **+** button to create a new configuration from a template.
+
+![Create Configuration](../../images/webstorm/new-configuration.png)
+
+In the list of templates, select **npm**.
+
+![Create Configuration](../../images/webstorm/select-npm-template.png)
+
+This creates a default npm configuration.
+
+![Create Configuration](../../images/webstorm/configuration.png)
+
+## Step 4 - Specify Configuration Options
+
+Select the npm script (`debug`) in the **Scripts** field.
+
+![Select Debug](../../images/webstorm/select-debug.png)
+
+### WebStorm version prior to 2018.3
+
+Specify the path to the test file you wish to debug in the **Arguments** field.
+
+![Enter File Path](../../images/webstorm/file-path-entered.png)
+
+Change this setting later to debug a different file.
+
+### WebStorm version 2018.3 and newer
+
+Click the **+** button in the **Arguments** field to insert a macro.
+
+![Click Add Macro Button](../../images/webstorm/click-add-macro-button.png)
+
+This invokes the **Macros** dialog.
+
+Select `FilePath` to insert the path to the current file.
+
+![Select Macro](../../images/webstorm/select-macros.png)
+
+Click **Insert**. A macro `$FilePath$` is inserted to the **Arguments** field.
+
+![Enter Macro](../../images/webstorm/macros-entered.png)
+
+This allows WebStorm to debug the currently opened test file.
+
+Click **OK** to save changes.
+
+## Step 5 - Debug Tests
+
+Open the file you wish to debug and put a breakpoint into it.
+
+Select the debug configuration you have created and click **Debug**.
+
+![Press Debug](../../images/webstorm/press-debug.png)
+
+TestCafe runs the test and stops at the breakpoint.
+
+![Stop on Breakpoint](../../images/webstorm/stop-on-breakpoint.png)
+
+Now you can use WebStorm debugging tools to inspect the code, step over to the next command, etc.
+
+> The debugger may consider TestCafe actions called in a chain as a single command. To correctly step between actions, you may need to add `await t.` before each of them.
```

Comment:
Or you can a breakpoint to each action of a chain.

Reply:
`> you may need to add 'await t' before each of them.`
It doesn't seem like the good idea.